packages/odyssey-react-labs/src/datePickerTheme.tsx

```diff
@@ -61,12 +62,12 @@ const todayDotStyles: StateStyles = {
   default: ({ theme }) => ({
     backgroundColor: theme.palette.primary.main,
     borderRadius: "50%",
+    bottom: theme.spacing(1),
     content: '" "',
-    height: `${(2 / 16) * (16 / 14)}rem`,
+    height: `${(2 / 16) * (16 / 14)}em`,
```

Comment:
Is this `2px` in Figma? If so, can prolly leave it as such here. The `(16 / 14)` modifier won't make sense if someone rethemes the base `fontSize`, since the proportional adjustment will be different.

If we do want it to scale with the `font-size`, the calculated value might be better since it won't need to reference the global values at all. It's something Design should probably create a token for anyway.

Reply:
`width: `${(2 / {theme.typography.fontSize})}rem`